__tests__/app.test.js

```diff
@@ -172,5 +172,45 @@ describe("app", () => {
           expect(res.body.msg).toBe("Bad Request");
         });
     });
+
+    test("POST: 201 - should add a comment to the relevant article_id", () => {
+      const newComment = {
+        body: "I am a comment!",
+        article_id: 1,
+        author: "butter_bridge",
+        votes: 0,
+        created_at: new Date(),
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          const comment = body.comment;
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.body).toBe("string");
+          expect(comment.article_id).toBe(1);
+          expect(typeof comment.author).toBe("string");
+          expect(comment.votes).toBe(0);
+          expect(typeof comment.created_at).toBe("string");
+        });
```

Comment:
As we know what new comment has been added, better to be specific about the body and the author as well